buildtest/cli/__init__.py

```diff
@@ -953,6 +953,9 @@ def report_menu(subparsers):
         action="store_true",
         help="Print output in machine readable format",
     )
+    parser_report.add_argument(
+        "--color", type=str, action="append", help="change the color of the output font"
```

Comment:
can we add a `choices` field see https://docs.python.org/3/library/argparse.html#choices and call a method that returns a list of colors.